draft-normington-jsonpath-latest.txt

```diff
@@ -246,16 +246,16 @@ Internet-Draft   JavaScript Object Notation (JSON) Path   September 2020
 
    Next, ".a" matches any input node of type object and selects any
    value of the input node corresponding to the key ""a"".  The result
-   is again a list of one node: "[{"b":0},{"b":1},{"b":2}]".
+   is again a list of one node: "[{"b":0},{"b":1},{"c":2}]".
 
    Next, "[*]" matches any input node which is an array and selects all
    the elements of the input node.  The result is a list of three nodes:
-   "{"b":0}", "{"b":1}", and "{"b":2}".
+   "{"b":0}", "{"b":1}", and "{"c":2}".
 
-   Finally, ".b" matches any input node of type object and selects any
-   value of the input node corresponding to the key ""b"".  The result
-   is the concatenation of three lists each of length one containing
-   "0", "1", "2", respectively.
+   Finally, ".b" matches any input node of type object with a key "b"
+   and selects the value of the input node corresponding to that key.
+   The result is the concatenation of two lists each of length one
```

Comment:
In fact, there are three lists concatenated: two of length one and one of length zero.

Reply:
Good catch. Fixed.